### PR TITLE
Upper Upper East Norfair - Gravity Lava Frames

### PR DESCRIPTION
**Pull request by kjbranch**
Partially addresses #663 
- Cathedral
- Norfair Reserve Tank Room
- Spiky Acid Snakes Tunnel

### CODE REVIEW

File: region/norfair/east.json [hunk 1]
```diff
@@ -4883,6 +4926,7 @@
                     "h_canNavigateHeatRooms",
                     "Gravity",
                     {"heatFrames": 375},
+                    {"lavaFrames": 100},
                     {"spikeHits": 2},
                     {"or": [
                       "canIframeSpikeJump",
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/norfair/east.json:4932]: I ran it with jumping immediately after each spike hit (not using iframes), and with good jumps and iframe runs. Jumping immediately takes fewer lava frames per jump, but the room is slower. Running in the lava for a bit takes more lava per jump. It just happened to work out to be the same for me

File: region/norfair/east.json [hunk 2]
```diff
@@ -878,16 +884,40 @@
...
+                          "Charge+Spazer",
+                          "PseudoScrew"
+                        ]
+                      }}
+                    ]}
                   ]
                 }
               ],
-              "note": "FIXME Apparently there's a way to get this without Morph?"
+              "devNote": "FIXME Apparently there's a way to get this without Morph?"
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/norfair/east.json:920]: I just left the note in there and changed it to a devNote, but I think I'll just delete it

File: region/norfair/east.json [hunk 3]
```diff
@@ -878,16 +884,40 @@
...
+                    ]},
+                    {"or": [
+                      {"enemyDamage": {
+                        "enemy": "Geruta",
+                        "type": "contact",
+                        "hits": 1
+                      }},
+                      {"enemyKill":{
+                        "enemies": [ ["Geruta"] ],
+                        "explicitWeapons": [
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/norfair/east.json:902]: Makes sense. I added any method that could kill it in 2 shots, or 1 charge shot and 1 normal shot

File: region/norfair/east.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (osse101) [reply region/norfair/east.json:4977]: Looking at what was here before PR though, the extra spike hits path is going to be slower.
And I don't see a damageboost strat here either.

[the remark this answers is not in the document]
Comment (kjbranch) [reply region/norfair/east.json:4977]: Ah, my bad. I thought we were talking about lava frames. The heat frames probably will be different. I'll look into that.

File: region/norfair/east.json [hunk 4]
```diff
@@ -831,7 +838,44 @@
...
+                      }},
+                      {"obstaclesCleared": ["A"]}
+                    ]}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        {"enemyKill":{
+                        "enemies": [ ["Geruta"] ],
```
Comment (osse101) [inline region/norfair/east.json:862]: indents

**Review by osse101 — DISMISSED** [review]
Just needs an update on the room diagram with the obstacle.

**Pull request merged**